src/OddJobs/Job.hs

```diff
@@ -456,7 +456,7 @@ jobMonitor = do
 
 -- | Ref: 'jobPoller'
 jobPollingSql :: Query
-jobPollingSql = "update ? set status = ?, locked_at = ?, locked_by = ?, attempts=attempts+1 WHERE id in (select id from ? where (run_at<=? AND ((status in ?) OR (status = ? and locked_at<?))) ORDER BY run_at ASC LIMIT 1 FOR UPDATE) RETURNING id"
```

Comment:
would this need an index on the `attempts` column, or it would not matter to the query plan much?

Reply:
I attempted adding an index on just attempts and an index on both attempts and run at, however it doesn't seem to want to use the indexes at all, using explain analyze on this query. The cost of sorting also seems really small.

Reply:
I guess there will need to be a very large number of jobs ready to be run for the index on `attempts` to be used.